packages/webkit2gtk_4.rb

```diff
@@ -56,22 +57,55 @@ class Webkit2gtk_4 < Package
   depends_on 'mesa'
   depends_on 'openjpeg'
   depends_on 'pango'
+  depends_on 'valgrind' => :build
   depends_on 'wayland'
   depends_on 'woff2'
   depends_on 'wpebackend_fdo'
 
+  def self.patch
+    open('CMakeLists.txt', 'a') do |f|
+      f.puts 'set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")'
+      f.puts 'set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")'
+    end
+  end
+
   def self.build
     # This builds webkit2gtk4 (which uses gtk3)
     Dir.mkdir 'builddir4'
     Dir.chdir 'builddir4' do
-      # -flto breaks x86_64 builds
-      # system "env #{CREW_ENV_OPTIONS} \
       # Bubblewrap sandbox breaks on epiphany with
       # bwrap: Can't make symlink at /var/run: File exists
-      # ccache currently breaks gcc builds of webkit-gtk
-      system "cmake \
+      case ARCH
+      when 'x86_64'
+        system "CC=clang CXX=clang++ LD=lld NM=llvm-nm cmake \
+          -G Ninja \
+          #{CREW_CMAKE_OPTIONS.gsub('-flto', '').gsub('-ffat-lto-objects', '')} \
```

Comment:
Stuff like this makes me cringe. :)

Reply:
Yeah me too. Easier to set defaults and back them out when they break something though?